content/en/building/cht-forms/_index.md

```diff
@@ -19,9 +19,9 @@ Forms are a building block of all CHT apps. They are used when creating or editi
 When a completed form is submitted, it is treated as a Report in the app. All reports can be viewed in the [Reports tab]({{< ref "building/features/reports" >}}) by those with the proper access within the [hierarchy]({{< ref "building/workflows/hierarchy" >}}).
 
 There are four different types of forms:
-- [**Contact Forms**]({{< ref "building/reference/forms/contact" >}}): used to create and edit contacts. Defined as CHT-enhanced XForms.
-- [**App Forms**]({{< ref "building/reference/forms/app" >}}): serve as actions within the app, such as a task or an action. Defined as CHT-enhanced XForms.
-- [**Collect Forms**]({{< ref "building/reference/forms/collect" >}}): used to render forms in Medic Collect. Defined as ODK XForms and need a corresponding JSON form to receive data in CHT.
+- [**Contact Forms**]({{< ref "building/cht-forms/forms/contact" >}}): used to create and edit contacts. Defined as CHT-enhanced XForms.
+- [**App Forms**]({{< ref "building/cht-forms/forms/app" >}}): serve as actions within the app, such as a task or an action. Defined as CHT-enhanced XForms.
+- [**Collect Forms**]({{< ref "building/cht-forms/forms/collect" >}}): used to render forms in Medic Collect. Defined as ODK XForms and need a corresponding JSON form to receive data in CHT.
 - [**JSON Forms**]({{< ref "building/reference/app-settings/forms" >}}): used for data coming from external channels such as SMS, or via interoperability with other tools. Defined according to a JavaScript Object Notation schema.
```

Comment:
as this is in a new section, and is actually about configuring forms, not about buildig forms, I think this shouldn't be here.  we have in related content already

```suggestion
```

Reply:
This bullet item is still in this file.  unresolving to track the request

(and if you don't want to do this, lemme know why)